node-require/calculate.js

```diff
@@ -0,0 +1,21 @@
+if (process.argv[3] === 'plus') {
+  const add = require('./add');
+  console.log('result:', add);
+}
+if (process.argv[3] === 'minus') {
+  const subtract = require('./subtract');
```

Comment:
Normally you would have all your `require` statements at the top of the file. It would bring in the `subtract` function and you would call subtract directly:
```
const subtract = require('./subtract'); // subtract is a function
subtract(n1, n2);